dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/KernelBootstrap.java

```diff
@@ -296,6 +296,27 @@ public ReturnCode shutdown() throws InterruptedException {
         return ReturnCode.OK;
     }
 
+    /**
+     * Force stop the server. This emulates the path that the server stop action takes:
+     * it calls shutdown on the LauncherDelegate, and then waits until it can obtain
+     * the server lock to ensure the server has stopped.
+     *
+     * @throws InterruptedException
+     */
+    public ReturnCode shutdown(boolean force) throws InterruptedException {
```

Comment:
Probably should have the no-arg shutdown method call this one, with a false argument. Something like:
```java
public ReturnCode shutdown() throws InterruptedException {
  return shutdown(false);
}
```